docker-compose.abuse.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  abuse:
+    build:
+      context: ./docker/abuse
+      dockerfile: Dockerfile
+    container_name: abuse
+    restart: unless-stopped
+    logging: *default-logging
+    env_file:
+      - .env
+    environment:
+      - ABUSE_LOG_LEVEL=${ABUSE_LOG_LEVEL}
+      - ABUSE_MAILBOX=${ABUSE_MAILBOX}
+      - ABUSE_SPONSOR=${ABUSE_SPONSOR}
+      - BLOCKER_API_URL=${BLOCKER_API_URL}
+      - BLOCKER_AUTH_HEADER=${BLOCKER_AUTH_HEADER}
+      - EMAIL_SERVER=${EMAIL_SERVER}
+      - EMAIL_USERNAME=${EMAIL_USERNAME}
+      - EMAIL_PASSWORD=${EMAIL_PASSWORD}
+      - MONGO_CONNECTIONSTRING=${MONGO_CONNECTIONSTRING}
```

Comment:
For this we also have env vars already - all of the `SKYNET_DB_*` ones.

While the `ACCOUNTS_EMAIL_URI` is only being used by `accounts` and we can change it if you really want, the DB ones are being used by a bunch of services by now (accounts, skyd, blocker, malware-scanner) and I don't think we'll change it.

Reply:
While reading the other PR about this service, I realised that this EMAIL_SERVER thing can be different from the email we want to use for sending emails. Let's rename these to something more descriptive, such as `ABUSE_EMAIL_SERVER` and so on, so we know right away (when reading the .env file) that this is the mailbox we want to watch for incoming abuse notifications and not the email account used for sending emails. Those can easily be different.

Reply:
I want to follow up with this one.